src/components/CardSeparator.tsx

```diff
@@ -10,7 +11,27 @@ import classnames, {
   width,
 } from 'classnames/tailwind'
 
-type Color = 'accent' | 'primary' | 'secondary' | 'tertiary' | 'transparent'
+type Gradient =
+  | 'secondary-to-transparent'
+  | 'accent-to-secondary'
+  | 'accent-to-transparent'
+interface CardSeparatorProps {
+  number: number
+  gradient: Gradient
+  vertical?: boolean
+  customHeight?: THeight
+}
+
+function getTailwindClassnames(gradient: Gradient) {
+  switch (gradient) {
+    case 'secondary-to-transparent':
+      return 'from-secondary'
```

Comment:
I found out, that tailwind applies transparent if we don't specify `to` or `from`, this is why I return only one string here